tests/reference/1b534d474638533530d8466363a16700.json

```diff
@@ -0,0 +1,450 @@
+{
+  "scr": {
+    "reg": "SCR",
+    "raw": "0205848700000000",
+    "fields": [
+      {
+        "field": "SCR_STRUCTURE",
+        "name": "SCR Structure",
+        "raw": [
+          0,
+          4
+        ],
+        "enum": "1.0"
+      },
+      {
+        "field": "SD_SPEC",
+        "name": "SD Memory Card - Spec. Version",
+        "raw": [
+          2,
+          4
+        ],
+        "enum": "2.00 or 3.0X"
+      },
+      {
+        "field": "DATA_STAT_AFTER_ERASE",
+        "name": "data status after erase",
+        "raw": [
+          0,
+          1
+        ],
+        "value": "False"
+      },
+      {
+        "field": "SD_SECURITY",
+        "name": "CPRM Security Support",
+        "raw": [
+          0,
+          3
+        ],
+        "enum": "No Security"
+      },
+      {
+        "field": "SD_BUS_WIDTHS",
+        "name": "DAT Bus widths supported",
+        "raw": [
+          5,
+          4
+        ],
+        "bits": [
+          "1 bit",
+          "4 bit"
+        ]
+      },
+      {
+        "field": "SD_SPEC3",
+        "name": "Spec. Version 3.00 or higher",
+        "raw": [
+          1,
+          1
+        ]
+      },
+      {
+        "field": "EX_SECURITY",
+        "name": "Extended Security Support",
+        "raw": [
+          0,
+          4
+        ]
+      },
+      {
+        "field": "RESERVED",
+        "raw": [
+          289,
+          9
+        ]
+      },
+      {
+        "field": "CMD_SUPPORT",
+        "name": "Command Support bits",
+        "raw": [
+          3,
+          2
+        ],
+        "bits": [
+          "Speed Class Control (CMD20)",
+          "Set Block Count (CMD23)"
+        ]
+      },
+      {
+        "field": "RESERVED_MFG",
+        "raw": [
+          0,
+          32
+        ]
+      }
+    ],
+    "computed": {}
+  },
+  "cid": {
+    "reg": "CID",
+    "raw": "1b534d474638533530d8466363a16700",
+    "fields": [
+      {
+        "field": "MID",
+        "name": "Manufacturer ID",
+        "raw": [
+          27,
+          8
+        ],
+        "enum": null
+      },
```

Comment:
Do you know the manufacturer of this card? Or is it one of the no-name unbranded varieties?

Reply:
It's a Samsung, which is not known to the decoder yet.